Pages/CourseList/script.js

```diff
@@ -1,54 +1,68 @@
-document.addEventListener('DOMContentLoaded', () => {
+import { baseUrl } from "../../config.js";
+const accessToken = localStorage.getItem("token");
+if (!accessToken) window.location.href = "../LoginPage/Index.html";
+
+document.addEventListener("DOMContentLoaded", () => {
     // Define the API endpoint
-    const apiEndpoint = 'http://devpass-backend-production.up.railway.app/api/v1/cursos/cursos';
 
     // Func. de dar fetch na API
     function fetchCourses() {
-        fetch(apiEndpoint)
-            .then(response => {
+        fetch(`${baseUrl}/cursos/cursos`, {
+            method: "GET",
+            headers: {
+                Accept: '"application/json"',
+                Authorization: `Bearer ${accessToken}`,
+            },
+        })
+            .then((response) => {
                 if (response.ok) {
+                    console.log("Response:  " + response);
                     return response.json();
                 } else {
-                    throw new Error('Network response was not ok');
+                    throw new Error("Network response was not ok");
                 }
             })
-            .then(data => {
-                displayCourses(data);
+            .then((data) => {
+                console.log("Data: ", data.cursos);
+                displayCourses(data.cursos);
             })
-            .catch(error => {
-                console.error('There was a problem with the fetch operation:', error);
+            .catch((error) => {
+                console.error(
+                    "There was a problem with the fetch operation:",
+                    error
+                );
             });
     }
 
     // Manipulação DOM
     function displayCourses(courses) {
-        const videosContainer = document.querySelector('.videos-container');
+        const videosContainer = document.querySelector(".videos-container");
 
         // Tirar o que tem no HTML
-        videosContainer.innerHTML = '';
+        videosContainer.innerHTML = "";
 
-        courses.forEach(course => {
+        courses.forEach((course) => {
```

Comment:
```suggestion
        courses.forEach((course) => {
```
evitar variável no plural, algo como courseList seria mais ideal até pra legibilidade de código